client/src/App.js

```diff
@@ -1,9 +1,12 @@
 import React from "react";
+import Signup from './pages/signUp';
+import './App.css'
+import 'bootstrap/dist/css/bootstrap.min.css'
```

Comment:
You should import it in the Signup page not here, and put your style in the end
```suggestion
import 'bootstrap/dist/css/bootstrap.min.css'
import './style.css'
```